gitdash/pages/api/all_repos.ts

```diff
@@ -0,0 +1,47 @@
+import { getSession } from "next-auth/client";
+import { Octokit } from "@octokit/core";
+import parse from "parse-link-header";
+
+export default async function GetDetails(
+  req: any,
+  res: {
+    status: (arg0: number) => {
+      (): any;
+      new (): any;
+      json: {
+        (arg0: {
+          repoNames: any;
+          user: any;
+          repoCount: any; 
+        }): any;
+        new (): any;
+      };
+    };
+  }
+): Promise<any> {
+  const session = await getSession({ req });
+
+  const octokit = new Octokit({
+    auth: session?.accessToken,
+  });
+
+  // Get user data
+  const userData = await octokit.request("GET /user");
+  const username = userData.data.login

```

Comment:
You should be consistent about whether or not your lines end in a semicolon `;`

Reply:
Python programmer messing up consistent semi-colons, my bad. I will fix this.